support-frontend/assets/helpers/abTests/abtestDefinitions.ts

```diff
@@ -90,6 +90,9 @@ export const tests: Tests = {
 			{
 				id: 'variant',
 			},
+			{
+				id: 'control',
+			},
```

Comment:
To double check this - would this enable the test? If so should we remove it and merge it later?

I'm assuming not as `isActive` is `false`.

Reply:
yeah, isActive will control whether the test is turned on or not ... control and variant are just the variants in the test.